contracts/truefi/TrueLender.sol

```diff
@@ -111,9 +112,32 @@ contract TrueLender is Ownable {
         pool.borrow(amount);
         currencyToken.approve(address(loanToken), amount);
         loanToken.fund();
+        loans.push(loanToken);
         emit Funded(address(loanToken), amount);
     }
 
+    function valueFor(ILoanToken loanToken) public view returns (uint256) {
+        (uint256 amount, uint256 apy, uint256 duration) = loanToken.getParameters();
```

Comment:
Mb it's worth adding a check that loan is funded and return 0 otherwise? It looks like unnecessary, but just in case logic changes somewhere in contract.

Reply:
And overall, checking for LT total supply looks like a good idea to avoid div by 0 exception

Reply:
Generally loan is added to the array when it's funded. I guess checking if the supply is not 0 should be enough.

Reply:
Better check if balance of not 0